multiqc_sarek_project.sh

```diff
@@ -58,7 +64,7 @@ fi
 # Gather a list of input dirs to give to MultiQC, exclude report directories not placed directly under the main sample
 # (i.e. reports for individual lanes etc. will be excluded)
 sed -nre 's/^.*<li>([^<]+)<\/li>.*$/\1/p' "$mqc_content/sample_list_mqc.yaml" > "$mqc_content/sample_names.txt"
-INPUT_DIRS="$mqc_content $(find "$PROJECT_PATH" -mindepth 3 -maxdepth 5 -type d -name "${PROJECT_ID}*" -a -path "*/Reports/*" |grep -f <(while read s; do echo "$s\$"; done < "$mqc_content/sample_names.txt") | paste -s -d' ')"
+INPUT_DIRS="$mqc_content $(find "$PROJECT_PATH" -mindepth 3 -maxdepth 5 -type d -name "${PROJECT_ID}*" -a -path "*/reports/*" | grep -f <(while read s; do echo "$s\$"; done < "$mqc_content/sample_names.txt") | paste -s -d' ')"
```

Comment:
This line is quite hard to wrap one's head around. I wonder if  it could be simplified like this: 
```suggestion
INPUT_DIRS=$(for sample in $(cat $mqc_content/sample_names.txt); do find "$PROJECT_PATH" -mindepth 3 -maxdepth 5 -type d -name $sample -a -path "*/reports/*"; done | paste -s -d' ')
INPUT_DIRS+=" $mqc_content $qc_content"
```
I think it would also be clearer to add that last line here instead of the change on line 90.

Reply:
Agree this is much clearer, however I changed it a bit since the INPUT_DIRS needs to be different for the two sbatch jobs.